Jenkinsfile

```diff
@@ -0,0 +1,13 @@
+@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+def dummy
+goTemplate{
+  dockerNode{
+      goMake{
```

Comment:
So they just want CI and they are not looking for CD, this pipeline should run on PR and also while the PR is merged to master. Hence I did not explicitly check if it is CI or CD.
@pradeepto can you please confirm this.

I am not sure if we still add this at least for the snapshot.
@rawlingsj